app/assets/javascripts/discourse/lib/search.js.es6

```diff
@@ -94,9 +94,9 @@ export function searchForTerm(term, opts) {
     };
   }
 
-  var promise = ajax('/search/query', { data: data });
+  let promise = ajax('/search/query', { data: data });
```

Comment:
BTW there is some disagreement in the JS world about whether to const everything or not. You've probably noticed that I prefer it. The idea is that it avoids bugs where you change things you didn't mean to. Also, in theory it should make JS faster because engines can optimize variables they know don't change.

In practice though, it stops very few bugs and even if we didn't transpile there's no engine where const is faster than let.

I still like it as a way of linting, but if someone prefers `let` I don't mind. As long as it's not `var` I'll accept it happily.

(I still prefer const myself)